insertion.py

```diff
@@ -0,0 +1,22 @@
+import time
+import random
+
+def tri_selection (tab: list[int]) -> list[int]:
+    i: int = 0
+    j: int = 1
+    k: int = 0
+    while j < len(tab):

```

Comment:
Ce premier while équivaut à une boucle for, qui serait plus simple !

Reply:
(en fait une boucle for est un while dont on connait à l'avance le nombre d'itérations, ce qui est ici le cas)